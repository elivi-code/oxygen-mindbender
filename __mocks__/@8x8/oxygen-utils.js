export * from '@8x8/oxygen-utils';

const useFontSize = jest.fn(() => 16);

export { useFontSize };
