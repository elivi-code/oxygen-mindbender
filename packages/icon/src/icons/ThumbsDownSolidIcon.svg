<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M9.23832 3.74456L6.21279 4.10935L3.25503 7.95041V12.3338L6.52651 15.4392L7.4097 15.9549L7.30209 16.6734C4.23214 15.7341 1.99951 12.878 1.99951 9.50049C1.99951 7.5094 2.7754 5.69952 4.04131 4.35672C5.30723 3.01391 7.06318 2.13818 9.02331 2.01536L8.99951 2.00049L17.4995 2.00049C18.8802 2.00049 19.9995 3.11978 19.9995 4.50049C19.9995 4.79189 19.9497 5.07164 19.858 5.33168C20.5402 5.76314 20.9995 6.5728 20.9995 7.50049C20.9995 7.81602 20.9464 8.1179 20.8494 8.39589C21.5412 8.84054 21.9995 9.61699 21.9995 10.5005C21.9995 11.3891 21.5359 12.1694 20.8373 12.6128C20.9421 12.8887 20.9995 13.1879 20.9995 13.5005C20.9995 14.8812 19.8802 16.0005 18.4995 16.0005L15.0173 15.5064C16.4509 18.5649 16.3861 20.7639 14.4002 21.7139C13.4024 22.1912 12.4792 22.0662 11.807 21.3702C11.4013 20.9501 11.2116 20.5535 10.8391 19.5728C10.5742 18.8755 10.4422 18.5997 10.3021 18.4546C9.75634 17.8894 8.79627 17.3043 7.42824 16.7281L7.28824 16.6692C4.22543 15.7255 1.99951 12.873 1.99951 9.50049C1.99951 7.5094 2.7754 5.69952 4.04131 4.35672C5.40893 2.90604 7.34847 2.00049 9.49951 2.00049L9.23832 3.74456Z" fill="#636363"/>
</svg>
