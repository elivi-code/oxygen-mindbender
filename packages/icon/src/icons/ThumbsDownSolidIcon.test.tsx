/* eslint-disable */
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { icon as theme } from '@8x8/oxygen-constants';
import ThumbsDownSolidIcon from './ThumbsDownSolidIcon';

describe('<ThumbsDownSolidIcon>', () => {
  let wrapper;

  function setup(props = {}) {
    return renderer.create(<ThumbsDownSolidIcon theme={theme} {...props} />).toJSON();
  }

  it('should render correctly', () => {
    wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
/* eslint-enable */
