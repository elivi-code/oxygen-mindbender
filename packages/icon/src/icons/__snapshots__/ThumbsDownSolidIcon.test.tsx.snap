// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ThumbsDownSolidIcon> should render correctly 1`] = `
.c0 {
  height: 24px;
  width: 24px;
  min-height: 24px;
  min-width: 24px;
  -webkit-user-select: auto;
  -moz-user-select: auto;
  -ms-user-select: auto;
  user-select: auto;
  pointer-events: auto;
  fill: #1C2025;
}

<svg
  className="c0"
  color={null}
  disabled={false}
  role="presentation"
  selected={false}
  size={null}
  viewBox="0 0 24 24"
>
  <g>
    <path
      d="m9.2383 3.7446-3.0255.3647L3.255 7.9505v4.3834l3.2715 3.1054.8832.5157-.1076.7185c-3.07-.9393-5.3026-3.7954-5.3026-7.173 0-1.991.7759-3.8009 2.0418-5.1437 1.266-1.3428 3.0219-2.2185 4.982-2.3413l-.0238-.015h8.5c1.3807 0 2.5 1.1194 2.5 2.5 0 .2915-.0498.5712-.1415.8313.6822.4314 1.1415 1.2411 1.1415 2.1688 0 .3155-.0531.6174-.1501.8954.6918.4446 1.1501 1.221 1.1501 2.1046 0 .8886-.4636 1.6689-1.1622 2.1123.1048.2759.1622.5751.1622.8877 0 1.3807-1.1193 2.5-2.5 2.5l-3.4822-.4941c1.4336 3.0585 1.3688 5.2575-.6171 6.2075-.9978.4773-1.921.3523-2.5932-.3437-.4057-.4201-.5954-.8167-.9679-1.7974-.2649-.6973-.3969-.9731-.537-1.1182-.5458-.5652-1.5058-1.1503-2.8739-1.7265l-.14-.0589c-3.0628-.9437-5.2887-3.7962-5.2887-7.1687 0-1.9911.7759-3.801 2.0418-5.1438C5.409 2.906 7.3485 2.0005 9.4995 2.0005l-.2612 1.744Z"
    />
  </g>
</svg>
`;
